### PR TITLE
cp, grub-install, grub-mkconfig, hexdump, readelf: add Chinese translation

### PR DESCRIPTION
**Pull request by errorcode7**
- [ ] The page (if new), does not already exist in the repository.
- [ ] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [ ] The page has 8 or fewer examples.
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [ ] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages.zh/linux/grub-install.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# grub-install
+
+> 安装GRUB到设备.
```
Comment (blueskyson) [inline pages.zh/linux/grub-install.md:3]:
```suggestion
> 安装 GRUB 到设备.
```

File: pages.zh/linux/grub-install.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+> 更多信息: <https://www.gnu.org/software/grub/manual/grub/html_node/Installing-GRUB-using-grub_002dinstall.html>.
+
+- 安装GRUB到基于BIOS的系统:
```
Comment (blueskyson) [inline pages.zh/linux/grub-install.md:6]:
```suggestion
- 安装 GRUB 到基于 BIOS 的系统:
```

File: pages.zh/linux/grub-install.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+`grub-install --target={{i386-pc}} {{path/to/device}}`
+
+- 安装GRUB到基于UEFI的系统:
```
Comment (blueskyson) [inline pages.zh/linux/grub-install.md:10]:
```suggestion
- 安装 GRUB 到基于 UEFI 的系统:
```

File: pages.zh/linux/grub-install.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+`grub-install --target={{x86_64-efi}} --efi-directory={{path/to/efi_directory}} --bootloader-id={{GRUB}}`
+
+- 安装预置指定模块的GRUB:
```
Comment (blueskyson) [inline pages.zh/linux/grub-install.md:14]:
```suggestion
- 安装预置指定模块的 GRUB:
```

**Comment by blueskyson** [comment]
Please put a space between Chinese words and Latin words. You can reference Chinese style-guide [#5952](https://github.com/tldr-pages/tldr/pull/5952).

File: pages.zh/linux/grub-install.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
+# grub-install
+
+> 安装 GRUB 到设备。
+> 更多信息：<https://www.gnu.org/software/grub/manual/grub/html_node/Installing-GRUB-using-grub_002dinstall.html>。
+
+- 安装 GRUB 到基于 BIOS 的系统：
+
+`grub-install --target={{i386-pc}} {{path/to/device}}`
```
Comment (navarroaxel) [inline pages.zh/linux/grub-install.md:8]: Please translate the arguments for get this easier for non-English speakers. Translate `path/to/device` and the other examples.

File: pages.zh/linux/grub-install.md [hunk 6]
```diff
@@ -0,0 +1,16 @@
+# grub-install
+
+> 安装 GRUB 到设备。
+> 更多信息：<https://www.gnu.org/software/grub/manual/grub/html_node/Installing-GRUB-using-grub_002dinstall.html>。
```
Comment (marchersimon) [inline pages.zh/linux/grub-install.md:4]:
```suggestion
> 更多信息：<https://www.gnu.org/software/grub/manual/grub/html_node/Installing-GRUB-using-grub_002dinstall.html>.
```
After Latin text, we use Latin punctuation

File: pages.zh/linux/grub-install.md [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+
+- 安装 GRUB 到基于 BIOS 的系统：
+
+`grub-install --target={{i386-pc}} {{path/to/device}}`
+
+- 安装 GRUB 到基于 UEFI 的系统：
+
+`grub-install --target={{x86_64-efi}} --efi-directory={{path/to/efi_directory}} --bootloader-id={{GRUB}}`
+
+- 安装预置指定模块的 GRUB ：
```
Comment (marchersimon) [inline pages.zh/linux/grub-install.md:14]:
```suggestion
- 安装预置指定模块的 GRUB:
```

File: pages.zh/linux/readelf.md [hunk 8]
```diff
@@ -0,0 +1,20 @@
+# readelf
+
+> 显示 EFI 文件信息。
+> 更多信息：<http://man7.org/linux/man-pages/man1/readelf.1.html>。
```
Comment (marchersimon) [inline pages.zh/linux/readelf.md:4]:
```suggestion
> 更多信息：<http://man7.org/linux/man-pages/man1/readelf.1.html>.
```

File: pages.zh/linux/grub-install.md (hunk 5 above)
Comment (CleanMachine1) [reply pages.zh/linux/grub-install.md:8]: Also maybe not device, use `block_device` since that what they are called.

Comment (marchersimon) [reply pages.zh/linux/grub-install.md:8]: But that should be done in the original page, before we change any translations

File: pages.zh/linux/grub-install.md (hunk 7 above)
Comment (errorcode7) [reply pages.zh/linux/grub-install.md:14]: all line is chinese  endwith a english character `:` , it seem to be strange.

File: pages.zh/linux/grub-install.md (hunk 5 above)
Comment (errorcode7) [reply pages.zh/linux/grub-install.md:8]: anyone using grub-install，must kown  what is `device`, we  assess that they know device like `/dev/sdx`,if they want kown detail info,they would refers to archlinux wiki .

File: pages.zh/linux/grub-install.md (hunk 6 above)
Comment (errorcode7) [reply pages.zh/linux/grub-install.md:4]: understand!

File: pages.zh/linux/grub-install.md (hunk 7 above)
Comment (marchersimon) [reply pages.zh/linux/grub-install.md:14]: If you'd like to know the reasoning behind this, check out #5240. But basically, we use the "style" of the character immediately before it.

File: pages.zh/linux/grub-install.md (hunk 5 above)
Comment (marchersimon) [reply pages.zh/linux/grub-install.md:8]: It would still be great if you could translate as many arguments as possible. Especially `path/to/device`.

Comment (errorcode7) [reply pages.zh/linux/grub-install.md:8]: in legacy bios,the grub will be install  on disk between sector2-sector62,so the `device` must be sdX which means disk,not sdX1 which means partion,should i translate `path/to/device` into `/dev/sdX` directly? or just in chinese?

File: pages.zh/linux/cp.md [hunk 9]
```diff
@@ -0,0 +1,32 @@
+# cp
+
+> 复制文件和目录。
+> 更多信息：<https://www.gnu.org/software/coreutils/cp>.
+
+- 复制一个文件到另外一个地方：
+
+`cp {{path/to/source_file.ext}} {{path/to/target_file.ext}}`
```
Comment (navarroaxel) [inline pages.zh/linux/cp.md:8]: Please translate the `path/to/...`  for get this easier for non-English speakers, in all pages.

**Review by marchersimon — APPROVED** [review]
Thanks @errorcode7

**Review by navarroaxel — APPROVED** [review]

**Comment by errorcode7** [comment]
@CleanMachine1 Is there anything else I need to modify?

**Review by CleanMachine1 — APPROVED** [review]

**Pull request merged**